--- a/create_user_functions.sql	2025-10-23 20:35:47
+++ b/create_user_functions.sql	2025-10-23 20:35:55
@@ -22,7 +22,7 @@
 )
 SELECT substr(string_agg(b, ''), 1, length) FROM bricks;
 $$
-SET search_path to 'pg_catalog';
+SET search_path = 'pg_catalog';
 
 CREATE OR REPLACE FUNCTION create_application_user(username text)
  RETURNS text
@@ -36,7 +36,7 @@
     RETURN pw;
 END
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION create_application_user(text) FROM public;
 GRANT EXECUTE ON FUNCTION create_application_user(text) TO admin;
@@ -55,7 +55,7 @@
     EXECUTE format($$ ALTER ROLE %I SET log_statement TO 'all' $$, username);
 END;
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION create_user(text) FROM public;
 GRANT EXECUTE ON FUNCTION create_user(text) TO admin;
@@ -72,7 +72,7 @@
     EXECUTE format($$ CREATE ROLE %I WITH ADMIN admin $$, rolename);
 END;
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION create_role(text) FROM public;
 GRANT EXECUTE ON FUNCTION create_role(text) TO admin;
@@ -95,7 +95,7 @@
     END IF;
 END
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION create_application_user_or_change_password(text, text) FROM public;
 GRANT EXECUTE ON FUNCTION create_application_user_or_change_password(text, text) TO admin;
@@ -113,7 +113,7 @@
     EXECUTE format($$ REVOKE admin FROM %I $$, username);
 END
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION revoke_admin(text) FROM public;
 GRANT EXECUTE ON FUNCTION revoke_admin(text) TO admin;
@@ -130,7 +130,7 @@
     EXECUTE format($$ DROP ROLE %I $$, username);
 END
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION drop_user(text) FROM public;
 GRANT EXECUTE ON FUNCTION drop_user(text) TO admin;
@@ -145,7 +145,7 @@
 AS $function$
 SELECT user_management.drop_user(username);
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION drop_role(text) FROM public;
 GRANT EXECUTE ON FUNCTION drop_role(text) TO admin;
@@ -160,7 +160,7 @@
 AS $function$
 SELECT pg_terminate_backend(pid);
 $function$
-SECURITY DEFINER SET search_path to 'pg_catalog';
+SECURITY DEFINER SET search_path = 'pg_catalog';
 
 REVOKE ALL ON FUNCTION terminate_backend(integer) FROM public;
 GRANT EXECUTE ON FUNCTION terminate_backend(integer) TO admin;
